koinex_db.js

```diff
@@ -11,7 +11,93 @@ var koinex_data = mongoose.Schema({
     collection: 'koinex_data',
     strict: true
 });
+
+var calculated_data = mongoose.Schema({
+    calculated: { type: Array },
+}, {
+    collection: 'calculated_data',
+    strict: false
+});
+
+var BTC = mongoose.Schema({
+    date: { type: String },
+    close: { type: String },
+    high: { type: String },
+    low: { type: String },
```

Comment:
these are also string? should they be decimal or something?

Reply:
yes sir but they are working fine. if you want me to change them, then i will change them